src/libs/ck-libs/NDMeshStreamer/DataItemTypes.h

```diff
@@ -2,6 +2,7 @@
 #define DATA_ITEM_TYPES_H
 
 #include <utility>
+#include "ckarray.h"
```

Comment:
What specifically do you need from ckarray.h? (list/state in comments). A pure Charm++ library shouldn't need that, but of course TRAM may need low level access.

Reply:
It was for `CkArrayIndex` in `DataItemHandle`. Actually it only needs `ckarrayindex.h` instead of `ckarray.h`, so fixed that and added a comment.